### ISSUE TITLE
Legionella pneumophila antigen in urine

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10713]
**Preferred term label:**
Legionella pneumophila antigen in urine

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C5878999

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Legionnaires' disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added

Positive urine Legionella pneumophila antigen test
HP:0025796
